crates/inline_completion_button/src/inline_completion_button.rs

```diff
@@ -0,0 +1,510 @@
+                        }
+                    }
+                    _ => IconName::CopilotInit,
+                };
+
+                if let Status::Error(e) = status {
+                    return div().child(
+                        IconButton::new("copilot-error", icon)
+                            .icon_size(IconSize::Small)
+                            .on_click(cx.listener(move |_, _, cx| {
+                                if let Some(workspace) = cx.window_handle().downcast::<Workspace>()
+                                {
+                                    workspace
+                                        .update(cx, |workspace, cx| {
+                                            workspace.show_toast(
+                                                Toast::new(
+                                                    NotificationId::unique::<CopilotErrorToast>(),
+                                                    format!("Copilot can't be started: {}", e),
+                                                )
+                                                .on_click("Reinstall Copilot", |cx| {
+                                                    if let Some(copilot) = Copilot::global(cx) {
+                                                        copilot
+                                                            .update(cx, |copilot, cx| {
+                                                                copilot.reinstall(cx)
+                                                            })
+                                                            .detach();
+                                                    }
+                                                }),
+                                                cx,
+                                            );
+                                        })
+                                        .ok();
+                                }
+                            }))
+                            .tooltip(|cx| Tooltip::text("GitHub Copilot", cx)),
+                    );
+                }
+                let this = cx.view().clone();
+
+                div().child(
+                    popover_menu("copilot")
+                        .menu(move |cx| {
+                            Some(match status {
+                                Status::Authorized => {
+                                    this.update(cx, |this, cx| this.build_copilot_context_menu(cx))
+                                }
+                                _ => this.update(cx, |this, cx| this.build_copilot_start_menu(cx)),
+                            })
+                        })
+                        .anchor(AnchorCorner::BottomRight)
+                        .trigger(
+                            IconButton::new("copilot-icon", icon)
+                                .tooltip(|cx| Tooltip::text("GitHub Copilot", cx)),
+                        ),
+                )
+            }
```

Comment:
I guess this introduced this bug https://github.com/zed-industries/zed/issues/11537